src/main/java/entity/player/ComputerPlayerFactory.java

```diff
@@ -0,0 +1,11 @@
+package entity.player;
+
+/**
+ * Factory for creating Computer Player objects.
+ */
+public class ComputerPlayerFactory implements PlayerFactory {
```

Comment:
Is there a need for these to be factories? I'm thinking of how Player objects will be implemented and used, and it would probably always involved explicitly instantiating an AI player of a Human player?